src/test/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/rca/integTests/tests/jvmsizing/resources/rca_master.conf

```diff
@@ -0,0 +1,94 @@
+{
+  "analysis-graph-implementor":
+    "com.amazon.opendistro.elasticsearch.performanceanalyzer.rca.store.AnalysisGraphTest",
```

Comment:
This doesn't matter at all for the IT right? @ARcaGraph will be taken over this?

Reply:
yeah.. that's right.